doc/vim-go.txt

```diff
@@ -1589,6 +1588,23 @@ By default "snakecase" is used. Current values are: ["snakecase",
 >
       let g:go_addtags_transform = 'snakecase'
 <
+                                                        *'g:go_fold_enable'*
+
+Control syntax-based folding. If it's 1 all syntax-based folds are enabled.
+You can enable specific fold regions by setting an array. Possible values are:
+
+- "block"       `{` .. `}` blocks.
+- "import"      `import` block.
+- "varconst"    `var` and `const` blocks.
+
+By default it's 1.
+>
+  let g:go_fold = 1
+<
+Enable folding of only imports:
+>
+  let g:go_fold = ['import']
+<
```

Comment:
Could also just make the default be `let g:go_fold = ['import', 'block', 'varconst']`? The `1` was just a shortcut for this.

I'm not sure if we need two variables, and it will confuse things if they're both set.